clientlibs/js/src/UpgradeClient.ts

```diff
@@ -61,6 +64,11 @@ export default class UpgradeClient {
         }
     }
 
+    async init(userId: string, group?: Map<string, Array<string>>, workingGroup?: Map<string, string>): Promise<Interfaces.IUser> {
```

Comment:
@YashilDepani I think we shouldn't anticipate userId in the argument because we already have that at the time of initializing  `UpgradeClient` class. We are also validating it in the function `this.validationClient` that if the userId exist or not so IMO, we can remove it from the argument.

@VivekFitkariwala  @nirmalpatel What are your thoughts here?

Reply:
Yes, I got that, so I instead use instantiated user id (this.userId) right?

Reply:
Right. But also need opinion of @VivekFitkariwala or @nirmalpatel here

Reply:
Code updated